### PR TITLE
Implement grpc protofile.

### PR DESCRIPTION
**Pull request by nbiryulin**

### CODE REVIEW

File: server/src/main/kotlin/com/handcontrol/server/grpc/HandleRequestImpl.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+@GRpcService
+class HandleRequestImpl : HandleRequestGrpc.HandleRequestImplBase() {
+
+    override fun login(request: Request.LoginRequest, responseObserver: StreamObserver<Request.LoginResult>) {
+        //do Authorize account here
+
+
+        //sent reply
+        val reply = Request.LoginResult.newBuilder().apply {
+            imei = request.imei;
```
Comment (nbiryulin) [inline server/src/main/kotlin/com/handcontrol/server/grpc/HandleRequestImpl.kt]: ; seems useless

File: server/src/main/kotlin/com/handcontrol/server/grpc/HandleRequestImpl.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        responseObserver.onCompleted()
+    }
+
+    override fun proRequest(request: Request.ClientRequets, responseObserver: StreamObserver<Request.ClientReply>) {
+        //save request into database here
+
+
+        //sent reply
+        val reply = Request.ClientReply.newBuilder().apply {
+            imei = request.imei;
```
Comment (nbiryulin) [inline server/src/main/kotlin/com/handcontrol/server/grpc/HandleRequestImpl.kt]: again

File: server/src/main/proto/com/handcontrol/server/request.proto [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+message LoginResult {
+  string imei = 1;
+  string message = 2;
+}
+
+message ClientRequets {
+  string imei = 1;
+  int32 ProId = 2;
```
Comment (nbiryulin) [inline server/src/main/proto/com/handcontrol/server/request.proto]: what is proId?

File: server/src/main/kotlin/com/handcontrol/server/Application.kt [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
 
-@SpringBootApplication
-class Application {
-    fun main(args: Array<String>) {
-        runApplication<Application>(*args)
+@SpringBootApplication(
+        exclude = [DataSourceAutoConfiguration::class]
+)
+open class Application{
+    @Bean
```
Comment (nbiryulin) [inline server/src/main/kotlin/com/handcontrol/server/Application.kt:15]: are u sure that it is the right way to initialize beans in main class?

File: server/src/main/proto/com/handcontrol/server/request.proto [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  rpc ProRequest(ClientRequets) returns (ClientReply);
+}
+message LoginRequest {
+  string imei = 1;
+  string login = 2;
+  string password = 3;
+}
+
+message LoginResult {
+  string imei = 1;
```
Comment (nbiryulin) [inline server/src/main/proto/com/handcontrol/server/request.proto]: why we need to return imei to mobile?

File: server/src/main/proto/com/handcontrol/server/request.proto (hunk 3 above)
Comment (pavlouha) [reply server/src/main/proto/com/handcontrol/server/request.proto]: prothesis Id, I guess

Comment (nbiryulin) [reply server/src/main/proto/com/handcontrol/server/request.proto]: ok

File: server/src/main/proto/com/handcontrol/server/request.proto [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  string message = 2;
+}
+
+message ClientRequets {
```
Comment (nbiryulin) [inline server/src/main/proto/com/handcontrol/server/request.proto]: why plural? everywhere single

File: server/src/main/proto/com/handcontrol/server/request.proto [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  string imei = 1;
+  int32 ProId = 2;
+  string request = 3;
```
Comment (nbiryulin) [inline server/src/main/proto/com/handcontrol/server/request.proto]: IMHO message, not request

File: server/src/main/proto/com/handcontrol/server/request.proto [hunk 8]
```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package com.handcontrol.server;
+
+service HandleRequest{
+  rpc login(LoginRequest) returns (LoginResult);
+  rpc ProRequest(ClientRequets) returns (ClientReply);
+}
+message LoginRequest {
```
Comment (nbiryulin) [inline server/src/main/proto/com/handcontrol/server/request.proto:8]: @Pavlouha one mobile has one proId ? should we pass it in login?

Comment (phuonganhngu) [reply server/src/main/proto/com/handcontrol/server/request.proto:8]: ProId is the id of prostheses part. I want to specific it.

File: server/src/main/proto/com/handcontrol/server/request.proto (hunk 6 above)
Comment (phuonganhngu) [reply server/src/main/proto/com/handcontrol/server/request.proto]: This is a typo

File: server/src/main/proto/com/handcontrol/server/request.proto (hunk 5 above)
Comment (phuonganhngu) [reply server/src/main/proto/com/handcontrol/server/request.proto]: i will delete it

File: server/src/main/kotlin/com/handcontrol/server/grpc/HandleRequestImpl.kt (hunk 2 above)
Comment (phuonganhngu) [reply server/src/main/kotlin/com/handcontrol/server/grpc/HandleRequestImpl.kt]: i will delete this one too

File: server/src/main/proto/com/handcontrol/server/request.proto (hunk 8 above)
Comment (nbiryulin) [reply server/src/main/proto/com/handcontrol/server/request.proto:8]: Yea , but I am thinking about how better to get info about protheseses and mobiles connected. Are they many to one, or one to one

**Pull request merged**